style.css

```diff
@@ -0,0 +1,153 @@
+*{
+    box-sizing: border-box;
+}
+body{
+    margin: 0%;
+    background: rgb(82, 78, 78);
+    font-weight: 500;
+}
+
+:root{
+    --background: rgba(0,214,170,.85);
+
+}
+header{
+    background: var(--background);
+    text-align: center;
+    position: fixed;
+    z-index: 1;
+    width: 100%;
+}
+nav{
+    position: absolute;
+    text-align: left;
+    top: 100%;
+    background: var(--background);
+    width: 100%;
```

Comment:
can we use here width: inherit; it will be more semantic  instead of percentage

Reply:
ok, thanks for suggestion.